src/js/wcmp-admin.js

```diff
@@ -511,6 +512,36 @@ jQuery(($) => {
     }
   }
 
+  /**
+   * Updates the package type link that opens the shipment options dialog with the new package type. Determined by given
+   *  select with package type options.
+   *
+   * @param {jQuery} packageTypeSelect
+   */
+  function updateShipmentOptionsDialogPackageType(packageTypeSelect) {
+    const element = packageTypeSelect[0];
+    const options = [...element.options].map(({value, innerText: label, selected}) => ({value, label, selected}));
```

Comment:
Waarom wordt element.options gedestruct?

Reply:
Omdat `element.options` een `HTMLOptionsCollection` is en die niet iterable is

Reply:
Kan je dan niet beter Array.from gebruiken?

Reply:
niet beter, wel mooier

Reply:
true that